templates/config/routes.rb

```diff
@@ -46,11 +48,9 @@
   get '/checkout/:state', to: 'checkout#edit', as: :checkout_state
   get '/checkout', to: 'checkout#edit', as: :checkout
 
-  get '/orders/populate', to: 'orders#populate_redirect'
```

Comment:
No replacement for that?

Reply:
Yes, Solidus is not using this route. It was added because people may refresh the browser page when adding to the cart in some weird way (maybe this is not more possible with recent browsers?) causing a 404 to happen. I think we can avoid this.